app/services/create_budget.rb

```diff
@@ -7,8 +7,8 @@ def initialize(activity:)
 
   def call(attributes: {})
     budget = Budget.new
-    budget.parent_activity = activity
     budget.assign_attributes(attributes)
+    budget.parent_activity = activity
```

Comment:
ICan this ever actually be a different activity passed in via the attributes if we're passing in the activity _and_ the id that was used to fetch it?

Reply:
I don't think my brain is working well enough to comprehend that question!